awx/ui/src/screens/Inventory/shared/Inventory.helptext.js

```diff
@@ -21,6 +21,8 @@ const ansibleDocUrls = {
     'https://docs.ansible.com/ansible/latest/collections/community/vmware/vmware_vm_inventory_inventory.html',
   constructed:
     'https://docs.ansible.com/ansible/latest/collections/ansible/builtin/constructed_inventory.html',
+  terraform:
+    'https://github.com/ansible-collections/cloud.terraform/blob/stable-statefile-inventory/plugins/inventory/terraform_state.py',
```

Comment:
Hi, this URL will be used in the help text for `Source variables` field so I think this should be the link for documentation for the plugin instead of the source code.

One of these is better choice I guess.

- https://github.com/ansible-collections/cloud.terraform/blob/main/docs/cloud.terraform.terraform_provider_inventory.rst
- https://galaxy.ansible.com/ui/repo/published/cloud/terraform/content/inventory/terraform_provider/

Reply:
@kurokobo the URLs you are suggesting are pointing to the `terraform_provider` inventory, but here we are using the `terraform_state` inventory plugin which is not yet released

Reply:
Oh, make sense. My lack of confirmation, sorry for annoying you😞 Ignore my comment, please.

Reply:
The URL was replaced `terraform_state` rst. #15241